### ISSUE TITLE
connect to a socket REPL

### ISSUE THREAD

**Issue by velkyel** [redguardtoo/js-comint#12]
is it possible to add connect to a socket REPL feature like in inf-clojure, please?
There is possible to define inf-clojure-program like cons pair ("localhost" . 5555).

**Comment by redguardtoo** [comment]
what's socket REPL?

**Comment by velkyel** [comment]
do you know emacs commint file?

**Comment by redguardtoo** [comment]
No. Do you mean comint? What's the advantage to start a socket REPL?

**Comment by velkyel** [comment]
comint, sorry. "socket REPL" is now known name?

**Comment by velkyel** [comment]
Advantage? Running game with embedded js vm. Toying, scripting, debugging.

**Comment by redguardtoo** [comment]
How can I do that in NodeJS? Can you show me some prototype?

**Comment by velkyel** [comment]
like this:
```
var net = require('net');

var server = net.createServer(function(socket) {
    socket.setEncoding('utf8');
    socket.on('data', function(data) {
        var res = eval(data);
        socket.end(res.toString());
    });
});

server.listen(8000);
```
and run-js localhost:8000 in emacs.

I don't know nodejs too much -- actually I'd like to use js-comint mode to connect to my running app with embedded duktape (duktape.org) vm. This app listen and eval incoming data like example server.
